### PR TITLE
Chore merge from 4b12084

### PR DESCRIPTION
**Pull request by julien-deramond**
- [x] https://github.com/twbs/bootstrap/commit/4b12084a2ab1216e8f7fd746cf6605fdc221177c
- [x] https://github.com/twbs/bootstrap/commit/1b39d1275bf5f91ff126faea33a6e6ab71ba991e
  - [x] **Upstream fix**: `js/test/unit/collapse.spec.js` should probably be updated (see https://github.com/twbs/bootstrap/pull/38089)
  - [x] Some specific fixes in Boosted can be found in `site/layouts/partials/footer.html` and `site/layouts/shortcodes/orange-footer.html`
  - [x] Add a note in the migration guide based on the feedback of the previous actions
- [x] https://github.com/twbs/bootstrap/commit/2cc580e2d1e70d942965d54365db24fe6728ad00
- [x] https://github.com/twbs/bootstrap/commit/212c0dfd076e4e504de135b2b41647199feb3696
- [x] https://github.com/twbs/bootstrap/commit/5a0899d0406379cef96a05a7e787bc45420da881
- [x] https://github.com/twbs/bootstrap/commit/d0b0fec4092ffe203adcccbe7892cdabef1b5a06
- [x] https://github.com/twbs/bootstrap/commit/dbdfab8a952bd0844d405eedeaf3ee2a0177bd9f
- ~~https://github.com/twbs/bootstrap/commit/0d64dc6dfc1984672b2c9bb53462323ff7c7ceb0~~: nothing to do because these modifications come from Boosted
- [x] https://github.com/twbs/bootstrap/commit/e60002b1fc9bb8a735b3161a3e5d29fbb6a0cbdc
- [x] https://github.com/twbs/bootstrap/commit/e30916b5993f5bb13d652b2597cf1a4ef8326da9
- [x] https://github.com/twbs/bootstrap/commit/19d547be3d31bc96a163f56237dd372ba8994d35
- [x] https://github.com/twbs/bootstrap/commit/3ee2a7549553a81e1efc47af53a85f515cc3eaf9
  - I've chosen deliberately to keep `--base00` and `--base01` in light mode for Boosted based on our previous work on it to avoid any regressions 
- [x] https://github.com/twbs/bootstrap/commit/9013c9884e53be23f58e00026035baa38f2a31a2

### CODE REVIEW

File: site/assets/scss/_syntax.scss [hunk 1]
```diff
@@ -27,22 +27,23 @@
 // TODO: these values will need to be modified in order to work with Boosted dark mode
 // Right now, values are from Bootstrap from `--base00` to `--baseOF` and the 4 remaining one are copied from Boosted's light mode
 @include color-mode(dark, true) {
-  --base00: #282c34;
+  // --base00: #282c34;
+  // --base01: #353b45;
   --base01: #353b45;
```
Comment (louismaximepiton) [inline site/assets/scss/_syntax.scss:32]: This one shouldn't appear uncommented right ?

File: site/content/docs/5.3/examples/form/index.html [hunk 2]
```diff
@@ -217,7 +217,7 @@ <h1 class="display-1">Become a reseller</h1>
   <h2 class="visually-hidden">Sitemap and information</h2>
   <div class="container-xxl footer-terms">
     <ul class="navbar-nav gap-md-3">
-      <li class="fw-bold">© Orange 2022</li>
+      <li class="fw-bold">© Orange 2023</li>
```
Comment (louismaximepiton) [inline site/content/docs/5.3/examples/form/index.html:220]: Cant' we use something like `{{< year >}}` or `{{ now.Format "2006" }}` ? As we do in `.md` and in `/shortcodes`?

**Review by louismaximepiton — COMMENTED** [review]
Few small things but looks already good to me

File: site/content/docs/5.3/migration.md [hunk 3]
```diff
@@ -46,6 +46,19 @@ Learn more by reading the new [color modes documentation]({{< docsref "/customiz
 
 ### Components
 
+- **Accordion**
+  - <span class="badge bg-warning">Warning</span> For accessibility reasons, having a `aria-labelledby` on the collapse element in accordions is not necessary. Please reflect these modifications into your websites.
```
[the remark this answers is not in the document]
Comment (julien-deramond) [reply site/content/docs/5.3/migration.md:50]: Do you have an idea of how to explain it better?

File: site/assets/js/snippets.js [hunk 4]
```diff
@@ -60,14 +59,13 @@
       toast.show()
     })
 
-  // Instantiate all toasts in a docs page only
+  // Instantiate all toasts in docs pages only
   const toastTrigger = document.getElementById('liveToastBtn')
   const toastLiveExample = document.getElementById('liveToast')
   if (toastTrigger) {
```
[the remark this answers is not in the document]
Comment (julien-deramond) [reply site/assets/js/snippets.js:66]: Fixed in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1842/commits/727cfb73e212ac10c9c262f3a91270d43377f48a

File: site/content/docs/5.3/components/toasts.md [hunk 5]
```diff
@@ -92,11 +92,11 @@ We use the following JavaScript to trigger our live toast demo:
 ```js
 const toastTrigger = document.getElementById('liveToastBtn')
 const toastLiveExample = document.getElementById('liveToast')
+
 if (toastTrigger) {
+  const toastBoosted = bootstrap.Toast.getOrCreateInstance(toastLiveExample)
```
[the remark this answers is not in the document]
Comment (julien-deramond) [reply site/content/docs/5.3/components/toasts.md]: Good catch 😱 Thanks. Fixed in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1842/commits/f7f66a1c2fa019956f909d16db96a78e6d6d483b.

File: site/assets/scss/_syntax.scss (hunk 1 above)
Comment (julien-deramond) [reply site/assets/scss/_syntax.scss]: Fixed in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1842/commits/76f5c7288b39954dc1075157f50a03afc691a708

File: site/content/docs/5.3/examples/form/index.html (hunk 2 above)
Comment (julien-deramond) [reply site/content/docs/5.3/examples/form/index.html]: Nice enhancement ✨ Seems to work well with `{{< year >}}`. Fixed in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1842/commits/a003c0b09a482f5e442c1968b892334e9f9c9728.

**Review by louismaximepiton — APPROVED** [review]
Feels great !

**Pull request merged**